src/createServer.js

```diff
@@ -1,8 +1,45 @@
 'use strict';
 
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  const server = http.createServer((req, res) => {
+    const { pathname } = new URL(req.url, `http://${req.headers.host}`);
+    const normalizedPathWay = pathname.replace('/file', '') || 'index.html';
```

Comment:
The logic here assumes that if the pathname is not '/file', it defaults to 'index.html'. This might not be the intended behavior if the request is for a different file. Consider revising this logic to handle different file requests appropriately.